packages/js-sdk/src/sandbox/sandboxConnection.ts

```diff
@@ -111,7 +111,11 @@ export class SandboxConnection {
   private readonly client: APIClient
 
   // let's keep opts readonly, but public - for convenience, mainly when debugging
-  protected constructor(readonly opts: SandboxConnectionOpts) {
+  protected constructor(readonly opts: SandboxConnectionOpts, protected createCalled: boolean) {
+    if (!createCalled) {
+      throw new Error('Cannot instantiate Sandbox directly, use a `create()` method instead.')
```

Comment:
I'd frame it as this
```js
throw new Error('Cannot instantiate Sandbox directly. Use `const ci = await CodeInterpreter.create()` to get a new code interpreter instance.')
```
what do you think?

Reply:
what about something like this?
```
Cannot instantiate sandbox directly, use a `.create()` method instead of calling `new` to get a new sandbox.
```